app/Resources/assets/scss/main.scss

```diff
@@ -144,6 +144,24 @@ body#homepage {
 // Page: 'login'
 // --------------------------------------------------
 body#login {
+    #login-users-help p {
```

Comment:
Generally speaking you're right. But in this case I'd like to maintain it to only apply these styles when we are in the login page (although I recognize that it looks difficult to imagine displaying this same message in other pages).